test_responses.py

```diff
@@ -664,6 +664,34 @@ def run():
     assert_reset()
 
 
+def test_session_cookies():
+    body = b"test callback"
+    status = 200
+    headers = {"set-cookie": "session_id=12345; a=b; c=d"}
+    url = "http://example.com/"
+
+    def request_callback(request):
+        return (status, headers, body)
+
+    @responses.activate
+    def run():
+        responses.add_callback(responses.GET, url, request_callback)
+        session = requests.Session()
+        resp = session.get(url)
+        assert resp.text == "test callback"
+        assert resp.status_code == status
+        assert "session_id" in resp.cookies
+        assert resp.cookies["session_id"] == "12345"
+        assert resp.cookies["a"] == "b"
```

Comment:
A `Set-Cookie: session_id=12345; a=b; c=d` header should create a `session_id` cookie with `a` and `b` attributes; it should not create `a` and `b` cookies.  See PR#285 for more detail and an alternative solution.

Reply:
Thanks for catching the issue. I'm all for your PR to be merged instead of this one so long as it resolves the long-standing issue.